noetic/release-arm64-build.yaml

```diff
@@ -15,8 +15,17 @@ notifications:
   - sloretz+buildfarm@openrobotics.org
   maintainers: true
 package_blacklist:
-  - fetch_drivers
-  - ros_ign_gazebo
+- fetch_drivers
+- libntcan
+- mocap_nokov
+- nao_meshes
+- pepper_meshes
+- picovoice_driver
+- prosilica_gige_sdk
+- ros_ign_gazebo
+- ros_speech_recognition
+- rosrt
+- skyway
```

Comment:
Was the indentation change here done in response to yamllint feedback or just convention?

Reply:
I changed it to match the indentation of the other file so I could yank added lines from one buffer to the other for packages that were failing on both platforms.